crates/typst-cli/src/args.rs

```diff
@@ -386,6 +386,12 @@ pub struct FontsCommand {
 #[derive(Debug, Clone, Parser)]
 pub struct FontArgs {
     /// Adds additional directories to search for fonts
+    ///
+    /// The environment variable `TYPST_FONT_PATHS` is used as a fallback if
```

Comment:
Right now it says 
```
      --font-path <DIR>
          Adds additional directories to search for fonts
          
          [env: TYPST_FONT_PATHS=]
```
so this feels redundant. Especially since the same would apply to TYPST_ROOT otherwise, so it would be inconsistent with this change set.

The separator thing is still relevant though.

Reply:
Thanks for your kind review. I have removed this. :heart: